tox.ini

```diff
@@ -35,11 +35,7 @@ deps=
     asn1crypto==1.4.0
     mistune==2.0.3
     pytz==2022.2.1
```

Comment:
let's split this env into packages, tests and framework

Reply:
done https://github.com/valory-xyz/open-autonomy/pull/1415/commits/1c131bf9d788d7a95bd189dc726293d34ee65eb0